openedx/core/djangoapps/credentials/utils.py

```diff
@@ -28,6 +29,13 @@ def get_user_credentials(user):
         log.warning('Credentials configuration is disabled.')
         return credentials
 
+    use_cache = credentials_config.is_cache_enabled
+    if use_cache:
+        cache_key = credentials_config.CACHE_KEY + '.' + user.username
```

Comment:
This strikes me as odd. I understand why you're doing this - one user should never see another user's credentials - but it seems strange (and potentially expensive) to cache data separately for each user. @jimabramson is this something to be concerned about?